.github/workflows/ci.yaml

```
@@ -10,6 +10,16 @@ jobs:
     - uses: actions/setup-go@v2-beta
       with:
         go-version: '^1.14'
+    - run: echo "::add-path::$(go env GOPATH)/bin"
     - run: go version
+    - run: go get github.com/mattn/goveralls
+      env:
+        GO111MODULE: off
+    - run: curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin ${GOLANGCI_RELEASE}
+      env:
+        GOLANGCI_RELEASE: v1.21.0
```

Could we bump to the latest release now we're at it? https://github.com/golangci/golangci-lint/releases/tag/v1.24.0

---

Please use suggestions for this, then I can easily apply :)
```suggestion
        GOLANGCI_RELEASE: v1.24.0
```